cmake/find/FindOpenSSL.cmake

```diff
@@ -305,42 +305,56 @@ if(OPENSSL_INCLUDE_DIR AND EXISTS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h")
     REGEX "^#[\t ]*define[\t ]+OPENSSL_VERSION_NUMBER[\t ]+0x([0-9a-fA-F])+.*")
 
   string(COMPARE EQUAL "${openssl_version_str}" "" _is_empty)
-  if(_is_empty)
-    message(
```

Comment:
why was this error message removed? If version is empty then this would be silently ignored if I'm reading this right

Reply:
ah, I've missed the `else()` branch when skimming over. Could you add a comment here and in the else branch like "parsing versions for OpenSSL v3.0 and newer" and "parsing versions before OpenSSL v3.0, like v1.1.1"

Reply:
Done!

Reply:
perfect :)